FordLabs/peoplemover
# Organize application and test folders

## Issue
Resolves N/A

## What was done
- [x] Moved the `Styleguide` folder from the `Applications` folder to the root and renamed it `Styles`
- [x] Moved the `Assets` folder from the `Applications` folder to the root
- [x] Moved the `Fonts` folder from the `Applications` folder to the `Assets` folder
- [x] Moved `PeopleMover.tsx`  from the `Applications` folder to a `PeopleMover` folder
- [x] Moved `ErrorPageTemplate.tsx`  from the `Applications` folder to a `ErrorPageTemplate` folder
- [x] Moved all remaining unit tests out of the `tests` folder to directly next to the file they are testing 
- [x] Moved `TestUtils.tsx` from the `tests` folder to the `Utils` folder

## How to test
1. Ensure the application works as it did before.
2. Ensure the unit tests all pass.
